### PR TITLE
Add new CLI command prestashop:api-client to create and remove an API Client

### LINKED ISSUE #37513: Command for creating an API Client
### Prerequisites

- [x] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [x] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

For UI Tests, we need a CLI Command which allows us to create an API client and return the secret of the created API Client 

### Expected behavior

_No response_

### Steps to reproduce

N/A

### PrestaShop version(s) where the bug happened

9.0.x

### PHP version(s) where the bug happened

8.1

### If your bug is related to a module, specify its name and its version

_No response_

### Your company or customer's name goes here (if applicable).

@PrestaShopCorp

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 9.0.x
| Description?      | Add new CLI command `prestashop:api-client` to create and remove an API Client
| Type?             | new feature
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI green and maybe QA by dev
| UI Tests          | ~
| Fixed issue or discussion?     | Fixes #37513
| Related PRs       | ~
| Sponsor company   | ~

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hi, thanks for this contribution!

I found some issues with the Pull Request description:

* The `branch` should be `develop`, `9.0.x` or `8.2.x`. ([Read explanation](https://devdocs.prestashop-project.org/9/contribute/contribution-guidelines/pull-requests/#branch))

Would you mind having a look at it? This will help us understand how interesting your contribution is, thank you very much!
### About linked issues
Please consider opening an issue before submitting a Pull Request:
* If it's a bug fix, it helps maintainers verify that the bug is effectively due to a defect in the code, and that it hasn't been fixed already.
* It can help trigger a discussion about the best implementation path before a single line of code is written.
* It may lead the Core Product team to mark that issue as a priority, further attracting the maintainers' attention.

(Note: this is an automated message, but answering it will reach a real human)

File: src/PrestaShopBundle/Resources/config/services_dev/bundle/command.yml [hunk 1]
```diff
@@ -63,3 +63,8 @@ services:
   PrestaShopBundle\Command\UpdateLicensesCommand:
     tags:
       - 'console.command'
+
+  PrestaShopBundle\Command\APIClientCommand:
+    autowire: true
+    tags:
+      - 'console.command'
```
Comment (FabienPapet) [inline src/PrestaShopBundle/Resources/config/services_dev/bundle/command.yml]:
```suggestion
    autowire: true
    autoconfigure: true
```

Comment (FabienPapet) [reply src/PrestaShopBundle/Resources/config/services_dev/bundle/command.yml]: Use autoconfiguration to remove those ugly tags :D

**Review by FabienPapet — DISMISSED** [review]

File: src/PrestaShopBundle/Resources/config/services_dev/bundle/command.yml (hunk 1 above)
Comment (jolelievre) [reply src/PrestaShopBundle/Resources/config/services_dev/bundle/command.yml]: Good point, the other ones will be cleaned later I tried once to force autowiring and autoconfigue on the whole file but there were some errors I didn't have time to fix But ultimately this all should be simplified

**Review by FabienPapet — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Review by tleon — APPROVED** [review]

**Pull request merged**